lib/knapsack_pro/repository_adapters/git_adapter.rb

```diff
@@ -41,7 +42,11 @@ def build_author
       private
 
       def git_commit_authors
-        `git fetch --shallow-since "1 month ago" >/dev/null 2>&1 && git shortlog --summary --email --since "one month ago"`
+        if ENV.fetch('CI', 'false') == 'true'
```

Comment:
```suggestion
        if ENV.fetch('CI', 'false') == 'true' || KnapsackPro::Config::Env.detected_ci
```

Nice idea with checking CI=true here. Additionally we could check if any CI server was detected by us with the `detected_ci` method.
https://github.com/KnapsackPro/knapsack_pro-ruby/blob/f667808c9f02f918fcadd5f535bbde76f67be383/lib/knapsack_pro/config/env.rb#L262
https://github.com/KnapsackPro/knapsack_pro-ruby/blob/f667808c9f02f918fcadd5f535bbde76f67be383/lib/knapsack_pro/config/ci/base.rb#L29
I'm not sure if all CI providers use the `CI` env variable.

Reply:
We can also use `.downcase` method because some CI providers use `True` as value.
https://www.appveyor.com/docs/environment-variables/

```ruby
ENV.fetch('CI', 'false').downcase == 'true'
```